### ISSUE TITLE
Request for Clarification in README

### ISSUE THREAD

**Issue by pa-0** [githubnext/monaspace#261]
Currently this repositories README states:

***
> [!NOTE]
> Variable fonts are not yet well-supported in VS Code, and it is not yet possible to mix multiple fonts. Stay tuned, we're talking with the VS Code team about it!

You must use the `editor.fontLigatures` setting to enable the various features (texture healing, ligatures, and character variants). The setting is a comma-separated list of font features to be enabled.
***

However, I noticed that a PR has been merged into vscode since this was written:

[**PR #153968: Support variable fonts**](https://github.com/microsoft/vscode/pull/153968).  

So does the warning in the README still hold true, or have circumstances improved with respect to variable font support?

**Comment by pa-0** [comment]
I take that back, I see that this README was actually published ~one year after that PR was merged.  Still.. 
# 🤞🏽

**Comment by geroldmeisinger** [comment]
you can define a variable width font in `editor.fontFamily` and they will be rendered. however tabs and vertical cursor movement is broken. see here https://github.com/microsoft/vscode/issues/108755

**Comment by pa-0** [comment]
thanks @geroldmeisinger

**Comment by idan** [comment]
Yeah, we're excited for variable typeface support in VS Code but it doesn't look like it's robust enough yet for us to remove the warning. We're aware of this and will be very excited on the day that we can just tell people to use the variable font builds because they open up so many nice typographical options.

Cheers!
